README.md

```diff
@@ -87,4 +87,11 @@ The smoke.py script needs to access a base-runtime compiler test resource direct
 
 ### docker-image-name - path to mock configuration file
```

Comment:
Something needs to be re-worked with the README. The way the documentation is currently structured, it still suggests 'docker-image-name' is an avocado parameter.